packages/ui/alerts/alerts.jsx

```diff
@@ -194,7 +194,9 @@ export class Alerts extends React.PureComponent {
             popAlert(componentChannel, id);
           }, [timeout]);
 
-          this.tickets.push(ticket);
+          this.clearExistingTicket(id);
```

Comment:
@ailincaibianca modify `componentWillUnmount` to account for the new `{ id, ticket }` structure in `this.tickets`:
```javascript
 componentWillUnmount() {
    document.removeEventListener("form-alerts:message", this.onWarningEvent);
    try {
      this.tickets.forEach((entry) => clearTimeout(entry.ticket));
    } catch {
      // nothing worth handling
    }
```